problem_goals=['goal1']
actual_goal="goal1"
goals_dict={"goal1": {"domain":'/home/alice/catkin_ws/src/PROPER_Sofar/proper_reasoning/goal1_domain.pddl', 
      "problem":'/home/alice/catkin_ws/src/PROPER_Sofar/proper_reasoning/prova_problem.pddl',
      "init":'/home/alice/catkin_ws/src/PROPER_Sofar/proper_reasoning/init_problem.pddl',
      "command":'./ff -p /home/alice/catkin_ws/src/PROPER_Sofar/proper_reasoning/ -o goal1_domain.pddl -f prova_problem.pddl',
      "folder":'/home/alice/catkin_ws/src/PROPER_Sofar/proper_reasoning/',
      "plan":"/home/alice/catkin_ws/src/PROPER_Sofar/proper_reasoning/plan.pddl",
      "ontology":"http://www.semanticweb.org/alice/ontologies/2023/10/goal1#",
      "path_onto":"/home/alice/",
      }}


map_emotion_AV_axis={
   "SA":"S",
   "SU":"H",
   "H":"H",
   "A":"A",
   "D":"A",
   "F":"A",
   "N":"N",
   
}